src/createServer.js

```diff
@@ -1,11 +1,16 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const { router: userRouter } = require('./routes/user.route.js');
+const { router: expensesRouter } = require('./routes/expenses.route.js');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  app.use('/users', express.json(), userRouter);
+  app.use('/expenses', express.json(), expensesRouter);
+
+  return app;
```

Comment:
you should initialize users and expenses as empty arrays on each server running to make it work properly